.github/workflows/master.yml

```diff
@@ -300,28 +300,67 @@ jobs:
         git config --global push.default simple
 
-    - name: Deploy docs
-      # Only run this on main repo (not forks) for commits and releases but not for
-      # nightly builds.
-      if: github.repository == 'allenai/allennlp' && github.event_name != 'schedule'
+    - name: Set target folders
       run: |
         if [[ $GITHUB_EVENT_NAME == 'release' ]]; then
-            DIRECTORY=${GITHUB_REF#refs/tags/};
+            echo "::set-env name=DOCS_FOLDER::${GITHUB_REF#refs/tags/}"
         else
-            DIRECTORY=master;
+            echo "::set-env name=DOCS_FOLDER::master";
         fi
+        echo "Will deploy docs to $DOCS_FOLDER"
 
+    - name: Stage docs
+      # Only run this on main repo (not forks) for commits and releases but not for
+      # nightly builds.
+      if: github.repository == 'allenai/allennlp' && github.event_name != 'schedule'
+      run: |
         # Checkout allennlp-docs to /allennlp-docs
         git clone git@github.com:allenai/allennlp-docs.git ~/allennlp-docs
         
         # Copy the generated docs to the checked out docs repo
-        rm -rf ~/allennlp-docs/$DIRECTORY/
-        mkdir -p ~/allennlp-docs/$DIRECTORY
-        cp -r site/* ~/allennlp-docs/$DIRECTORY
+        rm -rf ~/allennlp-docs/$DOCS_FOLDER/
+        mkdir -p ~/allennlp-docs/$DOCS_FOLDER
+        cp -r site/* ~/allennlp-docs/$DOCS_FOLDER
 
+    - name: Update shortcuts
+      # Only run this on main repo (not forks) for commits and releases but not for
+      # nightly builds.
+      if: github.repository == 'allenai/allennlp' && github.event_name == 'release'
+      run: |
+        cd ~/allennlp-docs/
+        echo "Updating latest/index.html"
+        mkdir -p latest
+        cat >latest/index.html << EOL
+        <!DOCTYPE html>
+        <html>
+          <head>
+            <meta http-equiv="Refresh" content="0; url=/${DOCS_FOLDER}/" />
+          </head>
+          <body>
+            <p>Please follow <a href="/${DOCS_FOLDER}/">this link</a>.</p>
+          </body>
+        </html>
+        EOL
+        cat latest/index.html
+
+        SUFFIX=$(echo $DOCS_FOLDER | sed -E 's/v[0-9]+\.[0-9]+\.[0-9]+//')
+        if [[ -z $SUFFIX ]] || [[ "$SUFFIX" == *"post"* ]]; then
+            # If there's no suffix to the version or the suffix is a 'post',
+            # then it's considered stable.
+            echo "Updating stable/index.html";
+            mkdir -p stable;
+            cp latest/index.html stable/index.html;
+            cat stable/index.html;
+        fi
+
+    - name: Deploy docs
+      # Only run this on main repo (not forks) for commits and releases but not for
+      # nightly builds.
+      if: github.repository == 'allenai/allennlp' && github.event_name != 'schedule'
+      run: |
         # And push them up to GitHub
-        cd ~/allennlp-docs/$DIRECTORY
-        git add .
+        cd ~/allennlp-docs/
+        git add -A
         git commit -m "automated update of the docs"
```

Comment:
You can also just do `git commit -a -m "automated update of the docs"`.

Reply:
@schmmd that would miss any new files added I believe

Reply:
Yep--you're likely right!